pypeit/sensfunc.py

```diff
@@ -255,12 +261,25 @@ def splice(self, wave):
             splice_wave_mask = (wave_splice >= wave_mask_min) & (wave_splice <= wave_mask_max)
             sensfunc_splice[splice_wave_mask] = self.eval_sensfunc(wave_splice[splice_wave_mask], idet)
 
+        # Interpolate over gaps
+        zeros = sensfunc_splice == 0.
+        if np.any(zeros):
+            msgs.info("Interpolating over gaps")
+            interp_func = scipy.interpolate.interp1d(wave_splice[np.invert(zeros)],
+                                                 sensfunc_splice[np.invert(zeros)],
+                                                 kind='nearest', fill_value='extrapoloate')
+            zero_values = interp_func(wave_splice[zeros])
+            sensfunc_splice[zeros] = zero_values
+
         self.steps.append(inspect.stack()[0][3])
 
         return wave_splice, sensfunc_splice
 
     def show(self):
-        pass
+        plt.figure()
```

Comment:
Do you need the `plt.figure()` and `plt.close()` lines?

Reply:
removed the `figure`.  Am pretty sure I need the `close`